modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,9 +1250,257 @@ endif::[]
 
-CAUTION: Remote recovery is not intended to be a fully-functional disaster recovery solution. It is only safe when no other clusters are still writing to the specified bucket or container. Recovery does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data.
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster 
+. Restore data from a failed source cluster to the new cluster
+. Perform a rolling restart, if necessary
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. 
+
+CAUTION: Whole cluster restore is not intended to be a fully-functional disaster recovery solution. It does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data, although it will reflect the latest available offset in the topic data that is being restored. Committed transactions are not guaranteed to be atomic.
+
+=== Start a target cluster
+
+For Redpanda Enterprise, follow the steps to https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/[deploy a new cluster] in a self-hosted environment.
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. 
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+=== Whole cluster restore
+
+Data from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk cluster storage restore start
+----
+
+The following metadata is included in the restore:
+
+- Topic definitions. If you have enabled tiered storage only for specific topics, topics without tiered storage enabled will not be restored 
+- Users and access control lists (ACLs)
+- Schemas
+- The xref:develop:consume-data/consumer-offsets.adoc[consumer offsets topic]
+- All cluster configurations, including your Redpanda license key
+- Transaction metadata, up to the highest committed transaction. In-flight transactions are treated as aborted and will not be included in the restore
+
+NOTE: If you need to restore only a subset of topic data, consider using <<remote-recovery,remote recovery>> instead of a whole cluster restore.
+
+The destination cluster must be empty; that is, the cluster cannot contain data other than node and cluster configuration properties. If metadata indicating usage is found in the destination (such as user-provided schemas, non-superusers, ACLs, consumer offsets, or user-created topics), the restore will fail and the rpk command will issue a warning.
```

Comment:
>If metadata indicating usage is found in the destination (such as user-provided schemas, non-superusers, ACLs, consumer offsets, or user-created topics), the restore will fail and the rpk command will issue a warning

@mattschumpert in the current implementation this isn't true. I opted towards making the operation retriable, meaning e.g. it's fine if the topic already exists, it just won't be restored and the restore will move on rather than failing. I can make the change to be more strict if that behavior is more intuitive.

Reply:
- Users that already exist aren't created again
- ACL bindings I believe are additive so repeated application is idempotent
- Offsets is a good question: right now it would override the existing offsets, which seems less palatable if users have already committed offsets with an existing group. An alternative would be to ignore the consumer group if it's already exists. Or we can fail the operation entirely -- I'm open to this, but just exploring of ways to make the restore as low-touch as possible, if possible.

Reply:
@andrwng in the current implementation will the restore still be considered "complete" when there's existing metadata and it gets overridden? Will the command issue some output or indication on what was overridden or ignored? And by "retriable" do you mean manually by the user, or does the process itself automatically retry?

Reply:
>An alternative would be to ignore the consumer group if it's already exists.

Discussed with @mattschumpert and I will make a change before the release. In this way, most metadata updates will be additive and effectively idempotent (though cluster config restore will overwrite existing cluster configs).

Let's still document that we strongly recommend that the starting state of the cluster before restoring _should_ be empty (no topics or anything, no on-going produces, consumes, commits, transactions, etc).